README.md

```diff
@@ -28,46 +28,94 @@ The zkEVM back-end is called `EraVM`, and the architecture is called `eravm`.
 
 The zkEVM LLVM framework must be built with our tool called `zkevm-llvm`:
 
-1. Install some tools system-wide:
-   1.a. `apt install cmake ninja-build clang-13 lld-13` on a Debian-based Linux, with optional `musl-tools` if you need a `musl` build
-   1.b. `pacman -S cmake ninja clang lld` on an Arch-based Linux
-   1.c. On MacOS, install the [HomeBrew](https://brew.sh) package manager (being careful to install it as the appropriate user), then `brew install cmake ninja coreutils`. Install your choice of a recent LLVM/[Clang](https://clang.llvm.org) compiler, e.g. via [Xcode](https://developer.apple.com/xcode/), [Apple’s Command Line Tools](https://developer.apple.com/library/archive/technotes/tn2339/_index.html), or your preferred package manager.
-   1.d. Their equivalents with other package managers
+<details>
+<summary>1. Install system prerequisites.</summary>
 
-2. [Install Rust](https://www.rust-lang.org/tools/install)
+   * Linux(Debian):
 
-   Currently we are not pinned to any specific version of Rust, so just install the latest stable build for your platform.
-   Also install the `musl` target if you are compiling on Linux in order to distribute the binaries:
-   `rustup target add x86_64-unknown-linux-musl`
+      ```shell
+      # Install the required packages using apt
+      apt install cmake ninja-build curl git libssl-dev pkg-config clang lld
+      ```
+   * Linux(Arch):
 
-3. Install the zkEVM LLVM framework builder:
+      ```shell
+      # Install the required packages using pacman
+      pacman -Syu which cmake ninja curl git pkg-config clang lld
+      ```
 
-   3.a. `cargo install compiler-llvm-builder` on MacOS, or Linux for personal use
-   3.b. `cargo install compiler-llvm-builder --target x86_64-unknown-linux-musl` on Linux for distribution
+   * MacOS:
 
-   The builder is not the zkEVM LLVM framework itself, but a tool that clones its repository and runs the sequence of build commands.
-   By default it is installed in `~/.cargo/bin/`, which is recommended to be added to your `$PATH`.
```

Comment:
I’m sorry to see these lines removed; that baffled me in my early Rust days.

Reply:
They are not removed, they are [here](https://github.com/matter-labs/era-compiler-llvm/pull/520/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R81).